packages/vm/tests/BlockchainTestsRunner.ts

```diff
@@ -117,6 +118,33 @@ export default async function runBlockchainTest(options: any, testData: any, t:
       // Update common HF
       common.setHardforkByBlockNumber(currentBlock.toNumber())
 
+      if (raw.transactionSequence) {
+        const parentBlock = await vm.blockchain.getIteratorHead()
+        const blockBuilder = await vm.buildBlock({
```

Comment:
What do we think of this? Ethereum/tests should test our VM, so we'd actually would like to use runBlock, but for these cases we use buildBlock... I don't have any idea how to run it with runBlock though, since we can't produce invalid blocks.

Reply:
seems reasonable, `buildBlock` uses `runTx` under the hood